src/views/tickets/orders.php

```diff
@@ -43,7 +43,7 @@
 
 	<form method="post">
 
-	<?php tribe_tickets_get_template_part( 'tickets/orders-rsvp' ); ?>
+	<?php require( 'orders-rsvp.php' ); ?>
```

Comment:
This seems wrong: themers should be able to override `tickets/orders.php` without also overriding other templates to make it work, but using `require` like this prevents that.

Is there a reason for doing it this way?